gridt/src/app/home/home.page.html

```diff
@@ -7,26 +7,9 @@
   </ion-toolbar>
 </ion-header>
 
-<!-- If the user is not enrolled for any movements yet, inform his/her about how to do so-->
-<ion-content paddinsearchTextg>
-  <ion-grid *ngIf="movements$ && (movements$ | async).length === 0">
-    <ion-row>
-      <ion-col size="12" size-sm="8" offset-sm="2" text-center>
-        <p>
-          You haven't subscribed to any movements yet, let's do so now!
-          <ion-icon name="bicycle-outline"></ion-icon>
-          <ion-icon name="star"></ion-icon>
-          Movements can be found below:
-          <ion-button href="/movements" color="success" expand="full" shape="round">
-            Click me!
-          </ion-button>
-        </p>
-      </ion-col>
-    </ion-row>
-  </ion-grid>
+<ion-content>
 
-  <!-- Can be rewritten as an if/else statement together with previous if-->
-  <ion-grid *ngIf="movements$ && (movements$ | async).length > 0">
+  <ion-grid *ngIf="movements$ && (movements$ | async).length > 0; else Tutorial">
```

Comment:
This means that if the user is subscribed to only 1 movement and then unsubscribes he will see the tutorial again. Is this desired behaviour? (Not a rethorical question.)

Reply:
Yes, it is intended this way, so that the home page is never empty. If the user decides to unsubscribe from all their movements, the tutorial just comes back, which will redirect to the movements page if finished.